apis/keda/v1alpha1/triggerauthentication_types.go

```diff
@@ -114,13 +117,14 @@ type PodIdentityProvider string
 // PodIdentityProviderNone specifies the default state when there is no Identity Provider
 // PodIdentityProvider<IDENTITY_PROVIDER> specifies other available Identity providers
 const (
-	PodIdentityProviderNone          PodIdentityProvider = "none"
-	PodIdentityProviderAzure         PodIdentityProvider = "azure"
-	PodIdentityProviderAzureWorkload PodIdentityProvider = "azure-workload"
-	PodIdentityProviderGCP           PodIdentityProvider = "gcp"
-	PodIdentityProviderSpiffe        PodIdentityProvider = "spiffe"
-	PodIdentityProviderAwsEKS        PodIdentityProvider = "aws-eks"
-	PodIdentityProviderAwsKiam       PodIdentityProvider = "aws-kiam"
+	PodIdentityProviderNone           PodIdentityProvider = "none"
+	PodIdentityProviderAzure          PodIdentityProvider = "azure"
+	PodIdentityProviderAzureWorkload  PodIdentityProvider = "azure-workload"
+	PodIdentityProviderGCP            PodIdentityProvider = "gcp"
+	PodIdentityProviderSpiffe         PodIdentityProvider = "spiffe"
+	PodIdentityProviderAwsEKS         PodIdentityProvider = "aws-eks"
+	PodIdentityProviderAwsEKSWorkload PodIdentityProvider = "aws-eks-workload"
```

Comment:
Why did you add this?

Reply:
to help specify if it workload they will be using.